server.js

```diff
@@ -1,13 +1,8 @@
 const express = require('express')
-var io = require('socket.io');
 var app = express()
 var server = require('http').Server(app)
-var io = io(server);
+var io = require('socket.io')(server);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/CryptoChat-dev/cryptochat/pullRequest?prid=7237621)
```suggestion
var io = require("socket.io")(server);
```